### DISCUSSION TITLE
Websockets connection failing on coolify

### DISCUSSION THREAD

**Discussion by ayeshamehta** [coollabsio/coolify#4369] [Q&A]
Hello,

I'm hosting a VPS on Hostinger. I installed Coolify to manage my projects.
I'm working on an express application which uses websockets.

When I run the application on locally from terminal / on docker, it works as expected.
When the application is deployed using coolify on my VPS, it's failing to connect to the websocket server.
I'm pulling my hairs to identify where I made a mistake. I would appreciate is someone can point me in the correct direction.

### Server Side
```
const wss = new WebSocket.Server({ port: 443 });
```

### Client Side
```
const currentHostname = window.location.hostname;
const socket = new WebSocket(`ws://${currentHostname}:443`);
socket.onerror = (error) => {
	console.error('Websocket error:', error);
};
```

### Dockerfile
```
FROM node:20-bullseye-slim

WORKDIR /

COPY package*.json ./
COPY . .

RUN npm install


EXPOSE 3000
EXPOSE 443

CMD ["npm", "start"]
```

### Coolify Ports Exposed
![coolify config](https://github.com/user-attachments/assets/0358d45d-8483-462f-ab03-76958dd83c2c)

**Comment by ayeshamehta** [comment]
I was able to figure it out. Closing this out.

**Reply by BradleyLund** [reply]
I'm having the same issue for a simple websockets app. What resolved it for you?
